### ISSUE TITLE
Error 527

### ISSUE THREAD

**Issue by InvisibleImp** [moonlight-stream/moonlight-android#299]
I keep getting error 527, saying it failed to start app and game launcher has failed on the server. It is an intermittent problem, if I reboot it might work it might not. I have been using this for a year and never had this problem. I have reinstalled geforce experience and moonlight, and am running Windows anniversary edition, please help I use this every day and it's really messing me up!

**Comment by minyan** [comment]
Same here, this is a problem with GFE v 3.X, for time being downgrade to prior version (GFE v2.11.X)

**Comment by ChazzaMason** [comment]
This happens to me to. I turn my computer on remotely using Wake-on-LAN, and this error appears when I try to start a game. However when I lock and unlock the computer after turning it on, everything works fine. Seems like a glitch with some service not starting until the computer is locked/unlocked or something.

**Comment by cgutman** [comment]
@ChazzaMason it looks like NVIDIA knows about that bug - see this forum post: https://forums.geforce.com/default/topic/972789/gamestream/gamestream-error-800821f4/

Hopefully they'll fix it in a GFE update.

**Comment by minyan** [comment]
OK, i solved the problem.
Turns out there is a mismatch between the DRIVER and the GFE version that i'm using.
At the time of previous wrote up, i use Driver v 373.06 with GFE 3.X
The latest one (at the time of this writing; v375.30 is disabling Shield preferences in GFE 2.X

So to solved it,:

1. Completely uninstall the Driver AND GFE.
2. Reboot
3. Download and do Clean Install of the latest Driver (v375.70)
4. Reboot
5. Set GFE Shield Preferences as usual.

However some games is not receiving input properly.

In short you have two option :  
Use Driver v373.06 and GFE v2.11.4.0 -> This one is work on all games i tested.
OR
Use Driver v375.70 and GFE 3.1.0.X ->Broken Input on some games.

In other word, if you intend to use GFE v2.11.4.0, you can't update the driver because it will broke Moonlight.

That's all of my report i hope it helps someone. Sorry if it turns out misleading/wrong info.

**Comment by Staubgeborener** [comment]
I can confirm, that the driver v373.06 with GFE v2.11.4.0 works fine, @minyan . GFE seems to have some issues with v.3.X.

The streaming quality is not the best (720p/60 fps or 1080p/30 fps), but it works. Still it doesn't look really like hd quality on my hdtv, but well.
